### PR TITLE
Fix/related devices release

### PR DESCRIPTION
**Pull request by hassan-yonomi**
improved syncing for related devices

### CODE REVIEW

File: src/utils/relatedDevice.js [hunk 1]
```diff
@@ -36,7 +42,17 @@ class RelatedDevice {
...
+  update(data){
+    return this.request.rpc('put', [{custom: data}])
+      .then(response => {
+        this._self = response.result.body;
+        Object.assign(this, this._self);
+        return response;
+      })
+  }
+
+  commission(opts){
```
Comment (mikelittman) [inline src/utils/relatedDevice.js]: Couldn't this be cleaned up as `commission(opts = {})`

File: src/utils/relatedDevice.js [hunk 2]
```diff
@@ -36,7 +42,17 @@ class RelatedDevice {
     };
   }
 
-  commission(){
+  update(data){
+    return this.request.rpc('put', [{custom: data}])
+      .then(response => {
+        this._self = response.result.body;
```
Comment (mikelittman) [inline src/utils/relatedDevice.js]: You don't alias `this` here, but below it is; which way is correct?

File: src/utils/relatedDevice.js [hunk 3]
```diff
@@ -91,15 +116,17 @@ class RelatedDevice {
       );
   }
 
+  sync(){
+    let _this = this;
```
Comment (mikelittman) [inline src/utils/relatedDevice.js]: Aliasing this?

**Review by mikelittman — APPROVED** [review]
👍💯

**Review by ghost — APPROVED** [review]

**Pull request merged**